package.json

```diff
@@ -13,18 +20,33 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@semantic-release/changelog": "^6.0.3",
-    "@semantic-release/git": "^10.0.1",
     "cloudinary": "^1.40.0",
+    "fs-extra": "11.1.1",
```

Comment:
i had tried to remove this as a dependency - do we need this here? what are we using it for?